src/app/static/src/app/components/privacyPage.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div class="container mt-5">
+        <div class="row">
+            <div class="col">
+                <h1>Privacy notice for mint.dide.ic.ac.uk</h1>
+                <p>
+                    In accordance with data protection legislation and the privacy electronic communications regulations, the privacy notice will:
+                </p>
+                <ul>
+                    <li>inform you as to how we look after your personal data when you visit our websites;</li>
+                    <li>tell you about your privacy rights;</li>
+                    <li>tell you how the law protects you;</li>
+                </ul>
+                <p>
+                  This policy is based on the <a href="https://www.imperial.ac.uk/about-the-site/privacy/" target="_blank" rel="noopener noreferrer">Imperial College privacy policy</a>.
+                </p>
+
+                <h2>What do we collect?</h2>
+                <p>If you use this application we will record:</p>
+                <ul>
+                    <li>Your IP address</li>
+                    <li>Browser version details</li>
+                    <li>Which areas of the website you visit</li>
+                </ul>
+                <p>No other personal data is kept on our servers.</p>
+
+                <h2>How we use your personal data</h2>
+                <p>
+                  We record this data to generate analytics reports on the usage of the site in order to improve it.
+                </p>
+
+                <h2>Where we store your data</h2>
+                <p>
+                  We store your data on our own servers based in London, UK
+                </p>
+
+                <h2>How we keep your data secure</h2>
+                <p>
+                  We have put in place appropriate security measures to prevent your personal data from being accidentally lost, used or
+                  accessed in an unauthorised way, altered or disclosed. In addition, we limit access to your personal data to the
+                  administrators of MINT, who have a business need to know, and are subject to a duty of confidentiality.
+                </p>
+
+                <h2>How long will we use your personal data for?</h2>
+                <ul>
+                    <li>
+                      We will retain your personal data indefinitely to fulfil the purposes we collected it for, including for the purposes
```

Comment:
do we not rotate the logs pretty quickly?

Reply:
Don't think so, we still seem to have some stuff from years ago on there. Maybe we should!